src/App.tsx

```diff
@@ -1,7 +1,31 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
```

Comment:
```suggestion
import { PureComponent } from 'react';

type State = {
  pressedKey: string;
};

export class App extends PureComponent<{}, State> {
```
What if we add some typization?
And also change the component type so it won't rerender on each keypress.